### PR TITLE
feat: blog post When you buy an NFT, what are you actually getting?

### PR DESCRIPTION
**Pull request by jchris**
Initial draft of this article. was at https://www.notion.so/protocollabs/When-you-buy-an-NFT-what-are-you-actually-getting-dbc31c3285a34076a85a338830d87601

### CODE REVIEW

File: packages/website/posts/2022-01-25-what-am-buying.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: 'When you buy an NFT, what are you actually getting?'
+description: Behind the glamour, NFTs are more like hyperlinks than like files.
+author: J Chris Anderson
+thumbnail: https://user-images.githubusercontent.com/87873179/146266410-dac03a35-dff7-461b-8096-254f04e5df04.jpg
+date: Jan 25, 2022
+tags:
+  - culture
+---
+
+When you buy an NFT, you are purchasing your name at the top of the list of holders. The value comes from the previous holders (and the link to the creator), more than the image. You are also buying the right to decide who is the next token holder. You could sell it or give it away.
```
Comment (alanshaw) [inline packages/website/posts/2022-01-25-what-am-buying.mdx:11]: > more than the image

Not always an image! Could be a video, music, text, etc. "asset" maybe?

Note: you refer to "the image" throughout...

File: packages/website/posts/2022-01-25-what-am-buying.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+NFTs are not about the scarcity of the linked image, they are a new type of link. A link that memorializes how it moves through a community, and becomes valuable because of its history. A link that gains its uniqueness from human interactions.
+
+There is enough power in this new kind of link, just because it tracks how it changed hands, to enable a market. A market of links!
+
+Where does the value come from?
+
+The authentic link to the creator and the community of buyers, is encoded in that cryptographic list of holders. Without it, the NFT loses value. In the extreme case, machines link to an unsuspecting artist’s image, and make money from uneducated buyers. As more people understand that what they’re buying is a link with the artist and community, not the image itself, the public will become more educated about what’s worth buying.
+
+Tools owe it to users to make validating the provenance of NFTs a core part of the experience. If my pfp is legendary because it used to be owned by a celebrity, I want the world to know. Those same tools will help devalue spam, and put guardrails to prevent accidentally purchasing stolen NFTs.
```
Comment (alanshaw) [inline packages/website/posts/2022-01-25-what-am-buying.mdx:23]: Should "pfp" be capitalised? IDK what it stands for...total n00b.

File: packages/website/posts/2022-01-25-what-am-buying.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+NFT.Storage is building a dashboard to help you evaluate the health of your NFTs.
```
Comment (alanshaw) [inline packages/website/posts/2022-01-25-what-am-buying.mdx:25]: Sold! ...but wait, tell me more!

File: packages/website/posts/2022-01-25-what-am-buying.mdx (hunk 1 above)
Comment (jchris) [reply packages/website/posts/2022-01-25-what-am-buying.mdx:11]: good point

File: packages/website/posts/2022-01-25-what-am-buying.mdx (hunk 2 above)
Comment (jchris) [reply packages/website/posts/2022-01-25-what-am-buying.mdx:23]: profile picture. I don't know why its lowercase https://medium.com/geekculture/what-do-pfp-mean-role-of-nfts-in-pfp-768aa57e1746

File: packages/website/posts/2022-01-25-what-am-buying.mdx (hunk 3 above)
Comment (jchris) [reply packages/website/posts/2022-01-25-what-am-buying.mdx:25]: In my notes it end with "Experience the NiftyScore project here." but we need to ship that first lol

File: packages/website/posts/2022-01-25-what-am-buying.mdx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+title: 'When you buy an NFT, what are you actually getting?'
+description: Behind the glamour, NFTs are more like hyperlinks than like files.
+author: J Chris Anderson
+thumbnail: https://user-images.githubusercontent.com/22168118/149502173-d50d7e98-29a9-4c53-9dbc-ed19ab0ae755.png
+date: Jan 25, 2022
+tags:
+  - culture
+---
+
```
Comment (dchoi27) [inline packages/website/posts/2022-01-25-what-am-buying.mdx:10]: Might be worth sticking a paragraph about what NFTs are at their core - a unique virtual asset that links to somewhere else - might contextualize the first paragraph for readers

File: packages/website/posts/2022-01-25-what-am-buying.mdx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">When you buy an NFT, what are you actually getting?</p>&mdash; index.html (@jchris) <a href="https://twitter.com/jchris/status/1485664899862708225?ref_src=twsrc%5Etfw">January 24, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The authentic link to the creator and the community of buyers, is encoded in that cryptographic list of holders. Without it, the NFT loses value. In the extreme case, machines link to an unsuspecting artist’s image, and make money from uneducated buyers. As more people understand that what they’re buying is a link with the artist and community, not the image itself, the public will become more educated about what’s worth buying.
+
+Tools owe it to users to make validating the provenance of NFTs a core part of the experience. If my pfp is legendary because it used to be owned by a celebrity, I want the world to know. Those same tools will help devalue spam, and put guardrails to prevent accidentally purchasing stolen NFTs. Over time, people will learn what makes a good NFT, and scammers will find another playground.
+
```
Comment (dchoi27) [inline packages/website/posts/2022-01-25-what-am-buying.mdx:28]: Maybe worth mentioning how this isn't much different than physical art - originals vs. prints vs. counterfeits?

**Review by dchoi27 — APPROVED** [review]
Love the post! Left some suggestions

**Pull request merged**